### PR TITLE
Setup devenv for project development environment.

### PR DESCRIPTION
**Pull request by sadjow**
Devenv is an excellent tool for standardizing our development environment for everyone contributing to this project. It also helps quickly set up all the necessary tools, facilitating the onboarding of new contributors.

### CODE REVIEW

**Comment by sadjow** [comment]
@hakenprog, could you try it out in your environment to see if it is a good choice for this JS package? I have this small gist to setup it: https://gist.github.com/sadjow/898b1023695b983c7a1c742e605de381

File: devenv.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  languages.javascript = {
+    enable = true;
+    yarn.enable = true;
+  };
+
+  enterShell = ''
+    git --version
+    node --version
+    yarn install
```
Comment (sestrella) [inline devenv.nix:14]: @sadjow I believe there is an option for this:

https://devenv.sh/reference/options/#languagesjavascriptyarninstallenable

**Review by sestrella — APPROVED** [review]
@sadjow I'm glad to see Nix is reaching other projects! I only left one small comment.

**Review by hakenprog — APPROVED** [review]
Looks Good!

**Comment by sadjow** [comment]
Thank you for the review @sestrella @hakenprog

**Pull request merged**